FitNesseRoot/DbFit/AcceptanceTests/CoreTests/StoredProcNoParams/content.txt

```diff
@@ -1,5 +1,9 @@
 !3 If there are no parameters, Execute Procedure needs just one row
 
+|Clean|
+|table|clean?|
+|users|true|
```

Comment:
The `Users` table is also populated by `CommonTests.BulkInsertWithReturning` so I'll also have to change this test to clean up after itself (as we want the `Clean`s at the end of the test pages).